storybookjs/storybook
# Add uber example to docs site

Issue: N/A

## What I did

Add React-Vis from Uber

## How to test

```
yarn bootstrap --docs
yarn docs:dev
open http://localhost:8000
```
Is this testable with jest or storyshots?

Does this need a new example in the kitchen sink apps?

Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
